### DISCUSSION TITLE
Support request for Nedis WIFIZB10WT and ZBSC10WT

### DISCUSSION THREAD

**Discussion by AlderForrest** [make-all/tuya-local#831] [Ideas]
Hello!

Tried to add Nedis Zigbee Climate sensor thru Nedis Zigbee GW with no success. I presume there is not available support with the GW, is there possible to get this to work some how?

Some intel I have found out about Climate Sensor: 
DPSs: 
1 = temperature °C (scale 10)
2 = humidity % (scale 10)
4 = battery charge % (scale 10)
  
Zigbee gateway;
https://nedis.com/en-us/product/security-safety/smart-home/sets/550724698/zigbee-gateway-40-devices-usb-powered-android-ios-white

Climate Sensor:
https://nedis.com/en-us/product/smart-home/climate/monitoring/550726063/smart-climate-sensor-zigbee-30-battery-powered-android-ios-white

**Comment by AlderForrest** [comment]
With BlueStacks method I extracted from XML some info:

"name":"Zigbee Gateway","productId":"fqedbhzrnzgpofhz"
and
"name":"Temperature Humidity Sensor","productId":"fie1dpkm"

**Comment by make-all** [comment]
Support was added in 2023.8.2
